server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -163,7 +164,7 @@ while read size result ;do
 
```

Comment:
I believe that lines 137-164 should be deleted for this version: 

- we are going to create an $incoming link to the unpacked directory, so creating an $incoming directory is going to cause problems - and I don't think the "please wait" message has been at all useful. 
- `pbench-dispatch` has checked the MD5, so I don't see a need to do it again here.